### ISSUE TITLE
--next build error "Unexpected Character '@'"

### ISSUE THREAD

**Issue by schadenn** [ionic-team/stencil#2112]
**Stencil version:**
```
 @stencil/core@1.9.0-9
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**
Whenever I start a build using the `--next` flag, the build will fail with the following error:
```
[ ERROR ]  Rollup: Parse Error: src/components/my-component/my-component.tsx:4:0
Unexpected character '@' (Note that you need plugins to import files
that are not JavaScript) at error
```

**Expected behavior:**
Build using `--next` flag should succeed (as does the build without the `--next` flag).

**Steps to reproduce:**

- Run Windows (not sure that this is the cause, but I guess you're mostly using Macs)
- `npm init stencil`  --> component
- Run `stencil build --next`
- See the error mentioned above being logged to console

**Other information:**

I see that this is not yet stable, but reading this line (`Currently the next compiler is passing all tests, but we'd like to get more real-world testing before making it the default.`) in the Changelog I figured I should report this...?

**Comment by adamdbradley** [comment]
I'm unable to get this error following your steps. Would you be able to create a minimal repo that can recreate this issue? Thanks

**Comment by schadenn** [comment]
@adamdbradley I don't think there'd be any difference to this: [https://github.com/ionic-team/stencil-component-starter](https://github.com/ionic-team/stencil-component-starter)

I just tried reproducing the error in a completely different environment (I raised the issue from work where we're using a custom Nexus etc.), but I reproduced it at my private machine at home working with `https://registry.npmjs.org`.

I also tried the newly released `1.9.0-10` version. Even without installing the additional "lazyRequired" modules (updated my intro post).

`npm list` output:
```
stencil-next-test@0.0.1 X:\js_projects\stencil-next-test
`-- @stencil/core@1.9.0-10
  `-- typescript@3.7.4
```

`npx stencil build --next` output:
```
λ npx stencil build --next

Your current version of Node is v10.10.0, however the recommendation is a minimum of Node LTS (https://github.com/nodejs/Release). Note that future versions of Stencil will eventually remove support for non-LTS Node versions.


[ WARN  ]  Your current version of Node is v10.10.0, however the recommendation is a minimum of Node LTS
           (https://github.com/nodejs/Release). Note that future versions of Stencil will eventually remove support for
           non-LTS Node versions.

[33:56.3]  @stencil/core v1.9.0-10 [NEXT]

[ WARN  ]  This is a prerelease build, undocumented changes might happen at any time. Technical support is not
           available for prereleases, but any assistance testing is appreciated.

[33:57.0]  build, stencilnexttest, prod mode, started ...
[33:57.0]  transpile started ...
[33:57.8]  transpile finished in 769 ms
[33:57.8]  copy started ...
[33:57.8]  generate lazy started ...
[33:57.8]  copy finished (0 files) in 60 ms
[33:57.9]  generate lazy finished in 62 ms

[ ERROR ]  Rollup: Parse Error: src/components/my-component/my-component.tsx:4:0
           Unexpected character '@' (Note that you need plugins to import files that are not JavaScript) at error
           (X:\js_projects\stencil-next-test\node_modules\@stencil\core\compiler\stencil.js:35150:26) at Module$1.error
           (X:\js_projects\stencil-next-test\node_modules\@stencil\core\compiler\stencil.js:39493:5) at tryParse$1
           (X:\js_projects\stencil-next-test\node_modules\@stencil\core\compiler\stencil.js:39402:12) at
           Module$1.setSource
           (X:\js_projects\stencil-next-test\node_modules\@stencil\core\compiler\stencil.js:39720:29) at
           Promise.resolve.catch.then.then.then
           (X:\js_projects\stencil-next-test\node_modules\@stencil\core\compiler\stencil.js:41990:14) at
           process._tickCallback (internal/process/next_tick.js:68:7)

      L4:  @Component({
      L5:    tag: 'my-component',

[33:57.9]  build failed in 872 ms
```
